### PR TITLE
Add Eamonn and give integration access

### PR DESCRIPTION
**Pull request by 1pretz1**

### CODE REVIEW

**Comment by kevindew** [comment]
We seem to have the incorrect spelling of his name here: mcmlroy.

I notice we went for the correct one in https://github.com/alphagov/govuk-aws-data/pull/551/files btw

I'd suggest we go for the correct spelling and if that means IT need to change his email then so be it as we don't want to let the misspelling propagate too far.

**Comment by 1pretz1** [comment]
Yep good point, the name has been corrected - @kevindew

File: modules/users/manifests/eamonnmcelroy.pp [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Creates the eamonnmcmrloy user
```
Comment (kevindew) [inline modules/users/manifests/eamonnmcelroy.pp]: Just got one last misspelling to fix here.

Comment (1pretz1) [reply modules/users/manifests/eamonnmcelroy.pp]: ARGHHH, Eamonn actually sent me this and seemingly spelled his name doubly wrong haha -'eamonnmcmrloy' -> 'eamonnmcelroy'

**Pull request merged**